### ISSUE TITLE
Support API V4

### ISSUE THREAD

**Issue by Seraf** [kstaniek/fiblary#4]
Hello,

Don't know if there's lot of changes, but Fibaro will soon release the V4 API : http://www.developer.fibaro.com/docs/fghc-restapi

Do you plan to support it ?

Thanks !

**Comment by kstaniek** [comment]
Yes, I'm gonna look at this and actually this module is prepared to add a new controller for v4 but the problem is that I'm not gonna upgrade my HC2 to 4.0 yet. I don't feel it is stable and don't want to screw up my existing setup.

**Comment by Seraf** [comment]
I totally agree ! I won't upgrade until it's stable (and I will wait positive feedbacks on forum).
It was just to know if you planned to maintain it with the new api :)

Thanks !
